### ISSUE TITLE
Accessibility issue: item labels

### ISSUE THREAD

**Issue by marceloeler** [SecUSo/privacy-friendly-shopping-list#38]
Hi. I've been doing some research on accessibility features for mobile applications and selected some applications to evaluate. There are some improvements I'd suggest for your applications so users with visual disabilities would be able to use it, specially if they are using screen readers such as talk back:

Main activity
- The button (+) to create a list has no label, i.e., a speakable text for the screen reader

New List
- Consider making clickable items larger. For example, the input field to write the shopping list name, Notes, Priority spinner, Deadling checkbox and statistics on/off button. Moreover, the contrast ratio of the on/off button should be increased.

New Product
- Consider making clickable items larger. The icon to open more fields (category, store) has no label. A blind user using a screen reader would not know what is it for. 

Showing the products of the list
- icon to show the details of the product has no label.

Those are some examples of improvements you could make to became your app more accessible. I refer to [android accessibility best practices](https://developer.android.com/guide/topics/ui/accessibility/apps.html). If you need some help, please let me know I have an academic testing tool that might be useful to test your app and report all accessibility issues you might want to take a look.

**Comment by Yonjuni** [comment]
Thanks for telling the suggestions. We will look into that.
